Js bridge added. Wallets list updated (#1)
ton-connect/wallets-list: README.md

  "about_url": "https://tonkeeper.com",
  "universal_url": "https://app.tonkeeper.com",
  "bridge_url": "https://bridge.tonkeeper.com",
  "js_bridge_key": "tonkeeper"
}
```

#### Descritpion
- `name`: name of your wallet. Will be displayed in the dapp.
- `image`: icon of your wallet. Will be displayed in the dapp.

> - `tondns`: (optionoal) will be used in the protocol later.

Reviewer: ```suggestion
- `tondns`: (optional) will be used in the protocol later.
```

> - `about_url`: info or landing page of your wallet. May be useful for TON newcommers.

Reviewer: ```suggestion
- `about_url`: info or landing page of your wallet. May be useful for TON newcomers.
```

> - `universal_url`: (optional) base part of your wallet universal url. [Larn more](https://github.com/ton-connect/docs/blob/main/bridge.md#universal-link)

Reviewer: ```suggestion
- `universal_url`: (optional) base part of your wallet universal url. Your link should support [Ton Connect parameters](https://github.com/ton-connect/docs/blob/main/bridge.md#universal-link)
```

> - `bidge_url`: (optional) url of your wallet's implementation of the TonConnect HTTP bridge. [Learn more](https://github.com/ton-connect/docs/blob/main/bridge.md#http-bridge)

Reviewer: ```suggestion
- `bridge_url`: (optional) url of your wallet's implementation of the [HTTP bridge](https://github.com/ton-connect/docs/blob/main/bridge.md#http-bridge).
```

> - `js_bridge_key`: (optional) if your wallet handles JS Bridge connection, specify js key of your bridge object for `window` object. [Learn more](https://github.com/ton-connect/docs/blob/main/bridge.md#js-bridge)

Reviewer: ```suggestion
- `js_bridge_key`: (optional) if your wallet handles [JS Bridge](https://github.com/ton-connect/docs/blob/main/bridge.md#js-bridge) connection, specify the binding for your bridge object accessible through `window`. Example: the key `"tonkeeper"` means the bridge can be accessed as `window.tonkeeper`.
```

If your wallet supports HTTP Bridge, you have to specify `universal_url` and `bidge_url`. 

> If your wallet supports JS Bridge, you have to specify `js_bridge_key`.

Reviewer: ```suggestion
If your wallet provides the JS bridge (e.g. as a browser extension), you should specify the `js_bridge_key`.
```

> If your wallet supports both bridges, you have to specify `universal_url`, `bidge_url` and `js_bridge_key`.

Reviewer: ```suggestion
If your wallet supports both bridges, you have to specify `universal_url`, `bridge_url` and `js_bridge_key`.
```
